msgraph-teams-notification/src/msgraph/teams/notification/TeamsNotifier.java

```diff
@@ -38,7 +39,16 @@ public class TeamsNotifier extends NewTaskAssignmentListener {
   public TeamsNotifier() {
     super(IWorkflowManager.instance());
     taskHandler(this::notifyGraph);
-    client = Ivy.rest().client(UUID.fromString("fb0c277e-35a3-481f-8f79-edca67ce1145"));
+    client = backendClient();
+  }
+
+  private static WebTarget backendClient() {
+    return new MsGraph().client()
+      .property(OAuth2Feature.Property.SCOPE, Ivy.var().get("teams-notification.permissions"))
```

Comment:
by providing a different scope: tokens leased for a single client, by the same session "in the name of a user" or "application" are not mixed

Reply:
Does it make sense to use teams-notification.permissions as scope? We could also just use a simple constant, right?

I'm fine with this solution, but when I read this code, then I'm asking myself, is this right?
```
.property(OAuth2Feature.Property.SCOPE, Ivy.var().get("teams-notification.permissions"))
```

Reply:
I could hardcode this payload yes; and set it to "User.Read Chat.Create Chat.ReadWrite". 
I copied this highly configurable approach from the normal Feature+Variables.impl for MS connector. But it's not really providing value here ... so I'll simplify it